.circleci/config.yml

```diff
@@ -218,6 +218,18 @@ jobs:
       - run:
           name: "Test: Build and Spin-up Django API Service"
           command: "cd tdrs-backend; docker-compose up -d --build"
+      - run:
+          name:  "Test: Execute Python Linting Test"
+          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"flake8 .\""
+      - run:
+          name: "Test: LocalStack S3 bucket is available"
```

Comment:
This step was added because the LocalStack script used to create the buckets for tests does not run until after all the healthcheck metrics return true. This can cause test failures if the tests execute before the script has created the buckets. This command will use the localstack container's `awslocal` command to check the existence of the bucket. It will make up to 10 attempts, waiting 5 seconds between each attempt. If the bucket is not available at the end of this step it will be marked as failed, indicating a separate issue preventing the bucket from being created.